samples/test/stitcher.test.js

```diff
@@ -84,15 +80,26 @@ before(() => {
     .forEach(line => {
       if (line.includes(`locations/${location}/slates/${slateIdPrefix}`)) {
         this.nextId = line.split('/').pop();
-        try {
-          execSync(
-            `node deleteSlate.js ${projectId} ${location} ${this.nextId}`,
-            {
-              cwd,
+        let createTime = this.nextId.split('-').pop();
+        createTime = parseInt(createTime);
+        if (
+          isNaN(createTime) === false &&
+          createTime < DATE_NOW_SEC - ONE_HOUR_IN_SEC
+        ) {
+          try {
+            execSync(
+              `node deleteSlate.js ${projectId} ${location} ${this.nextId}`,
+              {
+                cwd,
+              }
+            );
+          } catch (err) {
+            if (err.message.includes('NOT_FOUND')) {
```

Comment:
Does the error object have a status to check the http status code? Might be cleaner than checking the message.